src/algorithms/expected_consensus.md

```diff
@@ -329,15 +348,15 @@ The probability that two Tipsets with different blocks would have all the same t
 
 {{<label finality>}}
 ## Finality in EC
-TODO
+EC enforces a version of soft finality whereby all miners at round N will reject all blocks coming in prior to round N-F. For illustrative purposes, we can take F to be 500. While strictly speaking EC is a probabilistically final protocol, choosing such an F simplifies miner implementations and enforces a socially-bound finality at no cost to liveness in the chain.
```

Comment:
- [ ] "... will reject all blocks that fork off prior to round N-F ...". ("coming in prior" is ill defined)
- [ ] s/"socially bound"/"macroeconomic consensus" or something like that?